File: Iniciante/1131/1131.c
#include <stdio.h>
 
int main() {
 
    int gol_gre, gol_int, v_gre, v_int, e, jogos, x;
    
    v_gre = 0;
    v_int = 0;
    e = 0;
    jogos = 0;
    
    do {
        scanf("%d%d", &gol_int, &gol_gre);
        if (gol_int > gol_gre)
            v_int++;
        else if (gol_int < gol_gre)
            v_gre++;
        else
            e++;
        jogos++;
        printf("Novo grenal (1-sim 2-nao)\n");
        scanf("%d", &x);
    } while (x == 1);
    
    printf("%d grenais\n", jogos);
    printf("Inter:%d\n", v_int);
    printf("Gremio:%d\n", v_gre);
    printf("Empates:%d\n", e);
    
    if (v_int > v_gre)
        printf("Inter venceu mais\n");
    else if (v_int < v_gre)
        printf("Gremio venceu mais\n");
    else 
        printf("Nao houve vencedor\n");
 
    return 0;
}

File: Python/Iniciante/1131.py
# Problema 1131 - Beecrowd - Iniciante - Nível 1
# Inicialização de contadores de placar
vitorias_gremio = 0
vitorias_inter = 0
empates = 0

# Entrada de placar de gols usando map
inter, gremio = map(int, input().split(' '))

# Verificação de qual time ganhou para ajustar contadores
if gremio > inter:
    vitorias_gremio += 1
elif gremio < inter:
    vitorias_inter += 1
else:
    empates += 1

# While verificando se o usuário digitou 1 ou 2
while (continua:= int(input('Novo grenal (1-sim 2-nao)\n'))) == 1:
    
    # Entrada de placar de gols usando map
    inter, gremio = map(int, input().split(' '))
    
    # Verificação de qual time ganhou para ajustar contadores
    if gremio > inter:
        vitorias_gremio += 1
    elif gremio < inter:
        vitorias_inter += 1
    else:
        empates += 1

# Exibição dos placares na saída de dados
print(f'{empates + vitorias_inter + vitorias_gremio} grenais')
print(f'Inter:{vitorias_inter}')
print(f'Gremio:{vitorias_gremio}')
print(f'Empates:{empates}')

# Verificação de qual time ganhou mais e exibição de resultado
# Final na saída de dados
if vitorias_inter > vitorias_gremio:
    print('Inter venceu mais')
elif vitorias_inter < vitorias_gremio:
    print('Gremio venceu mais')
else:
    print('Nao houve vencedor')
